starter-dev-e2e/cypress/fixtures/org/graphql/empty/orgRepos.empty.graphql.json

```diff
@@ -0,0 +1,34 @@
+{
+  "data": {
+    "organization": {
+      "id": "MDEyOk9yZ2FuaXphdGlvbjIyODM5Mzk2",
+      "repositories": {
+        "nodes": [],
+        "pageInfo": {
+          "endCursor": "Y3Vyc29yOnYyOpK5MjAyMi0wNC0wNVQwMTo1NzoxMS0wNTowMM4cfhyN",
+          "startCursor": "Y3Vyc29yOnYyOpK5MjAyMi0wNi0wN1QxMjoxMzo0OS0wNTowMM4ZGOxK",
+          "hasNextPage": true,
```

Comment:
Probably in this response we'll want to return false here -- it doesn't break anything, but the server probably wouldn't send us a true here if there was no results in the first query